sidebar/cSideBarFactory.cpp

```diff
@@ -25,7 +25,7 @@ cSideBarFactory *cSideBarFactory::getInstance() {
 
 // construct a cSideBar
 cSideBar * cSideBarFactory::createSideBar(cPlayer * thePlayer) {
-	assert(&thePlayer);
+	//assert(&thePlayer);
```

Comment:
why disable assertions here?

Reply:
g++ alert me that the assert is always true so it's usefull. 

![Capture d’écran_2022-02-13_11-41-45](https://user-images.githubusercontent.com/99535165/153749567-5e87bb6c-2972-48e6-ace2-6b92cc4ddea5.png)